ttt.log

```diff
@@ -0,0 +1,80 @@
+2020-07-24 15:40:22,570 - com.bl.ttt.main - ERROR - pytest: reading from stdin while output is captured!  Consider using `-s`.
+Traceback (most recent call last):
+  File "/Users/billlanyon/Documents/dev/source/TTT/com/bl/ttt/main.py", line 38, in get_user_input
+    i = input(prompt).upper()
+  File "/opt/anaconda3/lib/python3.7/site-packages/_pytest/capture.py", line 700, in read
```

Comment:
Looks like you've commited ttt.log by mistake - add an entry in .gitignore we only want source code nothing else in git.

Reply:
ttt.log added to gitignore